zipline/pipeline/filters/filter.py

```diff
@@ -427,3 +427,12 @@ class Latest(LatestMixin, CustomFilter):
     Filter producing the most recently-known value of `inputs[0]` on each day.
     """
     pass
+
+
+class SingleAsset(CustomFilter):
+    inputs = []
+    window_length = 1
+    params = ['asset']
+
+    def compute(self, today, assets, out, asset):
+        out[:] = (assets == asset.sid)
```

Comment:
This probably shouldn't be a `CustomFilter`, since it's always computing a constant over the input.  I would just make this a direct subclass of Filter.